db.json

```diff
@@ -6750,15 +7289,16 @@
   "font/otf": {
     "source": "iana",
     "compressible": true,
-    "extensions": ["otf"]
+    "extensions": ["otf","ttf"]
   },
   "font/sfnt": {
-    "source": "iana"
+    "source": "iana",
+    "extensions": ["ttf","otf"]
   },
   "font/ttf": {
     "source": "iana",
     "compressible": true,
-    "extensions": ["ttf"]
+    "extensions": ["ttf","otf"]
```

Comment:
Why add .otf to this? My understanding is that .otf = font/otf and .ttf = font/ttf (which is what the database was prior to adding this source).